src/instructlab/model/backends/backends.py

```diff
@@ -136,7 +143,6 @@ def is_model_safetensors(model_path: pathlib.Path) -> bool:
     requires_files = {
         "config.json",
         "tokenizer.json",
-        "tokenizer.model",
```

Comment:
removing this causes test to break because this is the only file that the test checks for.